### PR TITLE
Import 2011 PCT layers. https://github.com/acteng/atip/issues/337

### PR DESCRIPTION
**Pull request by dabreegster**
Frontend PR coming next

### CODE REVIEW

File: layers/pct.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    run(
+        [
+            "wget",
+            "https://github.com/npct/pct-outputs-national/raw/master/school/lsoa/rnet_all.geojson",
+            "-O",
+            f"{tmp}/school.geojson",
+        ]
+    )
+
+    # The two trip purposes are split into different files, and neither feature
```
Comment (dabreegster) [inline layers/pct.py:26]: @robinlovelace-ate , ideally we'd have 6 properties on each feature (the 3 scenarios for commuting and then for school). But there's no straightforward way to join the two files based on `local_id` or the feature ID. Is that correct?

File: layers/pct.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    # ID nor the local_id property matches between them. So just output two
+    # separate files.
+
+    def fixProps(inputProps):
+        return {
+            "baseline": int(inputProps["bicycle"]),
```
Comment (dabreegster) [inline layers/pct.py:32]: @robinlovelace-ate, am I picking up the correct fields? That output repo doesn't have / link to definitions for things, so I'll compare to carbon.place later manually

**Review by Pete-Y-CS — APPROVED** [review]

**Comment by dabreegster** [comment]
Thanks for the review! I'll merge now and ask Robin next week for any changes to make

File: layers/pct.py (hunk 1 above)
Comment (Robinlovelace) [reply layers/pct.py:26]: Yes, no easy way to join the 2... We can pre-process them into a single route network but I think it makes sense to keep them separate, cycling to school is a different kettle of fish.

File: layers/pct.py (hunk 2 above)
Comment (Robinlovelace) [reply layers/pct.py:32]: Yep!

Comment (Robinlovelace) [reply layers/pct.py:32]: There are definitions burried in there...

Comment (Robinlovelace) [reply layers/pct.py:32]: Here are the codebooks: https://github.com/npct/pct-shiny/tree/master/regions_www/www/static/02_codebooks/commute

**Review by Robinlovelace — COMMENTED** [review]
Looks great to me!

**Pull request merged**